felix/dataplane/linux/endpoint_mgr.go

```diff
@@ -730,6 +761,28 @@ func wlIdsAscending(id1, id2 *proto.WorkloadEndpointID) bool {
 	return id1.OrchestratorId < id2.OrchestratorId
 }
 
+func (m *endpointManager) hasSourceSpoofingConfiguration(interfaceName string) bool {
+	_, ok := m.sourceSpoofingConfig[interfaceName]
+	return ok
+}
+
+func (m *endpointManager) updateRPFSkipChain() {
```

Comment:
Think you need an unconditional start-of-day call to `updateRPFSkipChain` to make sure that this object is always going to set up that chain.  Right now, it's only done when an endpoint transition happens.

Reply:
I used to initialize the chain in the static chains, but that wasn't particularly clean. I updated the endpoints manager to set a flag `rpfSkipChainDirty` (similar to `hostEndpointsDirty`) when the RPF chain needs updating, and it is initialized to true when the manager is created to set up the chain.